### PR TITLE
Minor text corrections

### PR DESCRIPTION
**Pull request by keflavich**
WIP - I'll comment when I've finished reading, but I'm trying to make granular changes so that can be accepted/rejected independently.

### CODE REVIEW

**Comment by pllim** [comment]
Do you mean to push more commits here as you read, or should we just merge this?

**Comment by keflavich** [comment]
I'm going to push more commits

**Comment by pllim** [comment]
OK, I converted this to draft PR for now. Please mark it as ready when you are ready. Thanks! 😄

**Comment by keflavich** [comment]
Minor note: S4.4.2 calls out specific individuals by name, while other sections do not; I think we should not invoke individual names here.

**Comment by keflavich** [comment]
S4.4.3: what is `2dfdr`?

**Comment by pllim** [comment]
@keflavich , please use Discussion to raise issues that won't be addressed by this PR so they don't get overlooked. Thanks!

File: src/ms.tex [hunk 1]
```diff
@@ -1051,16 +1054,17 @@ \subsection{Astropy affiliated packages}
 done a few times per year.) \texttt{photutils} \citep{photutils} released its
 first stable version, indicating that the API will change less frequently, and
 there have been several significant performance improvements. \texttt{ccdproc}
-\citep{ccdproc} also released a new major version, bringing better performance to
+\citep{ccdproc}  released a new major version, bringing better performance to
 some image combination operations. The \texttt{regions} package
-\citep{pyregions}, for manipulating ds9-style region definitions \citep{ds9},
-added new ways to manipulate regions and introduced new region types.
+\citep{pyregions}, for manipulating ds9-, CASA-, or FITS style region definitions \citep{ds9},
```
Comment (nmearl) [inline src/ms.tex:1059]:
```suggestion
\citep{pyregions}, for manipulating ds9-, CASA-, or FITS-style region definitions \citep{ds9},
```

File: src/ms.tex [hunk 2]
```diff
@@ -1051,16 +1054,17 @@ \subsection{Astropy affiliated packages}
...
+expanded visualization and interactive region editing capabilities.
 \texttt{reproject}'s \citep{reproject} major new feature is a function to align
 and co-add images to create a mosaic; better support for parallelization was
 also added. \texttt{specutils}, the package that defines containers for 1D and
-2D spectra \citep{specutils}, also had its first stable release and the addition
-of classes to read JWST data. \texttt{stingray} has also had a major performance
-overhaul. The package \texttt{gammapy} \citep{gammapy} also has major performance
+2D spectra \citep{specutils},  had its first stable release and the addition
+of classes to read JWST data. \texttt{stingray} has had a major performance
+overhaul. The package \texttt{gammapy} \citep{gammapy} has major performance
```
Comment (nmearl) [inline src/ms.tex:1067]:
```suggestion
overhaul. The package \texttt{gammapy} \citep{gammapy} has had major performance
```

File: src/ms.tex [hunk 3]
```diff
@@ -925,7 +925,9 @@ \subsection{Project funding} \label{sec:project-funding}
 DRAGONS projects. The project also received financial support from the Dunlap
 Seed Funding program at the University of Toronto to develop educational
 resources for scientific software packaging within the Learn Astropy
-framework. All of these successful proposals show that the Astropy Project
+framework.  The project also indirectly benefited from the NRAO ALMA Development
+Study program, which supported the grant ``Linking CASA to the astropy ecosystem''\foontnote{\url{https://science.nrao.edu/facilities/alma/science_sustainability/Spectral_Cube_and_Radio_Beam_Ginsburg.pdf}}.
```
Comment (adrn) [inline src/ms.tex:929]:
```suggestion
Study program, which supported the grant ``Linking CASA to the astropy ecosystem''\footnote{\url{https://science.nrao.edu/facilities/alma/science_sustainability/Spectral_Cube_and_Radio_Beam_Ginsburg.pdf}}.
```

**Review by adrn — APPROVED** [review]

**Pull request merged**